datanode/api/trading_data_v2.go

```diff
@@ -220,39 +220,13 @@ func (t *tradingDataServiceV2) ListLedgerEntries(ctx context.Context, req *v2.Li
 		return nil, fmt.Errorf("could not parse ledger entry filter: %w", err)
 	}
 
-	groupOptions := &sqlstore.GroupOptions{}
-	if req.GroupOptions != nil {
-		groupByAccountField := []entities.AccountField{}
-		for _, field := range req.GroupOptions.ByAccountField {
-			field, err := entities.AccountFieldFromProto(field)
-			if err != nil {
-				return nil, fmt.Errorf("could not parse account from group options: %w", err)
-			}
-			groupByAccountField = append(groupByAccountField, field)
-		}
-
-		groupByLedgerEntryField := []entities.LedgerEntryField{}
-		for _, field := range req.GroupOptions.ByLedgerEntryField {
-			field, err := entities.LedgerEntryFieldFromProto(field)
-			if err != nil {
-				return nil, fmt.Errorf("could not parse ledger entry from group options: %w", err)
-			}
-			groupByLedgerEntryField = append(groupByLedgerEntryField, field)
-		}
-
-		groupOptions = &sqlstore.GroupOptions{
-			ByAccountField:     groupByAccountField,
-			ByLedgerEntryField: groupByLedgerEntryField,
-		}
-	}
-
 	dateRange := entities.DateRangeFromProto(req.DateRange)
 	pagination, err := entities.CursorPaginationFromProto(req.Pagination)
 	if err != nil {
 		return nil, apiError(codes.InvalidArgument, fmt.Errorf("invalid cursor: %w", err))
 	}
 
-	entries, pageInfo, err := t.ledgerService.Query(leFilter, groupOptions, dateRange, pagination)
+	entries, pageInfo, err := t.ledgerService.Query(leFilter, dateRange, pagination)
 	if err != nil {
 		return nil, fmt.Errorf("could not query ledger entries: %w", err)
```

Comment:
Could this return an apiError instead?

Reply:
Addressed here https://github.com/vegaprotocol/vega/pull/6834/files#diff-a493a31fca9e62f51a5f1c9971fa04a87210e5ff399f4737d5f8a610c5eb17c8R230
Thanks!